embedded-kafka/pom.xml

```diff
@@ -66,5 +66,16 @@
             <artifactId>camel-kafka</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
```

Comment:
what for web?

Reply:
https://github.com/testcontainers/testcontainers-spring-boot/pull/304#discussion_r425697321